src/tigerbeetle/main.zig

```diff
@@ -65,19 +65,24 @@ pub fn main() !void {
 
     var command = cli.parse_args(&arg_iterator);
 
+    switch (command) {
+        inline else => |*args| {
+            if (@hasField(@typeInfo(@TypeOf(args)).Pointer.child, "log_debug")) {
+                if (args.log_debug) {
+                    log_level_runtime = .debug;
+                }
+            }
+        },
+    }
```

Comment:
I think this would be more explicit + readable:
```suggestion
    switch (command) {
        .version => {},
        inline else => |*args| {
            if (args.log_debug) {
                log_level_runtime = .debug;
            }
        },
    }
```

In particular, if we add a new subcommand, it means we would need to explicitly "opt out" of debug logs for it to compile.